### ISSUE TITLE
Crash on :undo

### ISSUE THREAD

**Issue by crater2150** [qutebrowser/qutebrowser#3665]
I recently had several crashes when trying to reopen a tab with `:undo`

```
Received signal 11 SEGV_MAPERR 000000000048                                                                                                                                                                                              
#0 0x7ff7d83aee96 <unknown>                                                                                                                                                                                                              
#1 0x7ff7d6fda526 <unknown>                                                                                                                                                                                                              
#2 0x7ff7d83af20c <unknown>                                                                                                                                                                                                              
#3 0x7ff7ee9bc7d0 <unknown>                                                                                                                                                                                                              
#4 0x7ff7d705a5e4 QtWebEngineCore::WebContentsAdapter::webContents()                                                                                                                                                                     
#5 0x7ff7d705c55d <unknown>                                                                                                                                                                                                              
#6 0x7ff7e985ba34 <unknown>                                                                                                                                                                                                              
#7 0x7ff7e984f95b QObject::event()                                                                                                                                                                                                       
#8 0x7ff7e22826cc QApplicationPrivate::notify_helper()                                                                                                                                                                                   
#9 0x7ff7e2289e84 QApplication::notify()                                                                                                                                                                                                 
#10 0x7ff7ddbab12e <unknown>                                                                                                                                                                                                             
#11 0x7ff7e981f5d8 QCoreApplication::notifyInternal2()                                                                                                                                                                                   
#12 0x7ff7e98792de QTimerInfoList::activateTimers()                                                                                                                                                                                      
#13 0x7ff7e9879aa1 <unknown>                                                                                                                                                                                                             
#14 0x7ff7e840bdc7 g_main_context_dispatch                                                                                                                                                                                               
#15 0x7ff7e840c000 <unknown>                                                                                                                                                                                                             
#16 0x7ff7e840c08c g_main_context_iteration                                                                                                                                                                                              
#17 0x7ff7e9879e3f QEventDispatcherGlib::processEvents()                                                                                                                                                                                 
#18 0x7ff7c99f0261 <unknown>                                                                                                                                                                                                             
#19 0x7ff7e981d80a QEventLoop::exec()                                                                                                                                                                                                    
#20 0x7ff7e9826b74 QCoreApplication::exec()                                                                                                                                                                                              
#21 0x7ff7ddba9c9b <unknown>                                                                                                                                                                                                             
#22 0x7ff7eec7e8dd _PyCFunction_FastCallDict                                                                                                                                                                                             
#23 0x7ff7eecefca2 <unknown>                                                                                                                                                                                                             
#24 0x7ff7eecf2857 _PyEval_EvalFrameDefault                                                                                                                                                                                              
#25 0x7ff7eeceedef <unknown>                                                                                                                                                                                                             
#26 0x7ff7eecefb85 <unknown>                                                                                                                                                                                                             
#27 0x7ff7eecf2857 _PyEval_EvalFrameDefault                                                                                                                                                                                              
#28 0x7ff7eeceedef <unknown>                                                                                                                                                                                                             
#29 0x7ff7eecefb85 <unknown>                                                                                                                                                                                                             
#30 0x7ff7eecf2857 _PyEval_EvalFrameDefault                                                                                                                                                                                              
#31 0x7ff7eeceedef <unknown>                                                                                                                                                                                                             
#32 0x7ff7eecefb85 <unknown>                                                                                                                                                                                                             
#33 0x7ff7eecf2857 _PyEval_EvalFrameDefault                                                                                                                                                                                              
#34 0x7ff7eecef7d6 <unknown>                                                                                                                                                                                                             
#35 0x7ff7eecefd2e PyEval_EvalCodeEx                                                                                                                                                                                                     
#36 0x7ff7eecefd5b PyEval_EvalCode                                                                                                                                                                                                       
#37 0x7ff7eed18be0 <unknown>                                                                                                                                                                                                             
#38 0x7ff7eed1afc1 PyRun_FileExFlags                                                                                                                                                                                                     
#39 0x7ff7eed1b10f PyRun_SimpleFileExFlags                                                                                                                                                                                               
#40 0x7ff7eed316ae Py_Main                                                                                                                                                                                                               
#41 0x55be60beeb30 <unknown>                                                                                                                                                                                                             
#42 0x7ff7ee616021 __libc_start_main                                                                                                                                                                                                     
#43 0x55be60beebea <unknown>                                                                                                                                                                                                             
  r8: 0000000000000000  r9: 000055bee1c90160 r10: 00007ff7ef2a5e70 r11: 000055be62eccee0
 r12: 000055be62ede0a0 r13: 000055be628f2650 r14: 000055be84703da0 r15: 00007ffda33afa80
  di: 000055bee20a0410  si: 000055bee1c90160  bp: 0000000000000030  bx: 000055bee1c90160
  dx: 0000000000000000  ax: 0000000000000030  cx: 00007ffda33af780  sp: 00007ffda33af758
  ip: 00007ff7d705a5e4 efl: 0000000000010246 cgf: 002b000000000033 erf: 0000000000000004
 trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000048
[end of stack trace]
```
Output of `qutebrowser --version`:
```
qutebrowser v1.1.1                                                                                                                                                                                                    
Git commit: 
Backend: QtWebEngine (Chromium 61.0.3163.140)

CPython: 3.6.4
Qt: 5.10.1
PyQt: 5.8.2

sip: 4.19.7
colorama: no
pypeg2: 2.15
jinja2: 2.10
pygments: 2.2.0
yaml: 3.12
cssutils: no
attr: 17.4.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebKitWidgets: no
pdf.js: no
sqlite: 3.22.0
QtNetwork SSL: LibreSSL 2.5.5

Style: QFusionStyle
Platform: Linux-4.14.14_1-x86_64-with-glibc2.3.4, 64bit
Linux distribution: void (void)
Frozen: False
Imported from /usr/lib/python3.6/site-packages/qutebrowser
Qt library executable path: /usr/lib/qt5/libexec, data path: /usr/share/qt5

Paths:
cache: /home/crater2150/.cache/qutebrowser
config: /home/crater2150/.config/qutebrowser
data: /home/crater2150/.local/share/qutebrowser
runtime: /tmp/.private/crater2150/qutebrowser

Uptime: 0:00:00
```

**Comment by jgkamat** [comment]
I think this is a duplicate of https://github.com/qutebrowser/qutebrowser/issues/3619 and is fixed in https://github.com/qutebrowser/qutebrowser/commit/d44ff5ba01bea65444b96a05eb5252a39b99824f.

I think this fix was backported to `1.1.2`, can you upgrade to the patch release and see if it fixes the crash for you?

**Comment by The-Compiler** [comment]
Duplicate of #3619
